java/java.hints/src/org/netbeans/modules/java/hints/errors/Utilities.java

```diff
@@ -3196,7 +3198,7 @@ public static void performRewriteRuleSwitch(JavaFix.TransformationContext ctx, T
         for (Iterator<? extends CaseTree> it = cases.iterator(); it.hasNext();) {
             CaseTree ct = it.next();
             TreePath casePath = new TreePath(tp, ct);
-            patterns.addAll(ct.getLabels());
+            patterns.addAll(ct.getLabels());            
```

Comment:
Nit: unnecessary change.

Reply:
Thanks for the input. Have made the changes.

Reply:
Seems the change is still there - the import above seems unnecessary, and this line seems to have some unnecessary trailing whitespaces.